src/Form.js

```diff
@@ -8,7 +8,13 @@ class Form extends Component {
   render() {
     return (
       <form>
-        <input className='search-bar' type="text" name="search" placeholder="search for a movie"></input>
+        <input 
+          className='search-bar' 
+          type="text" 
+          name="search" 
+          placeholder="search for a movie"
+          onChange={(e) => this.props.filterMovies(e.target.value)}
```

Comment:
This totally makes sense to me. No need to make this a 'controlled form' when its value is only going to be cleared by the user deleting text. Clean and simple!